### PR TITLE
Fix experiment biome; Add experiment scienceSubject

### PR DESCRIPTION
**Pull request by Croissong**
I fixed the experiment biome (KSC biomes work) and added a scienceSubject property.
Now you can check if the experiment in the current situation has been completed, how much science is left to gather and more.

I tested it at Kerbin and Minmus and also with DMagic Orbital Science experiments.

References #296

### CODE REVIEW

File: service/SpaceCenter/src/Services/Parts/Experiment.cs [hunk 1]
```diff
@@ -146,7 +146,20 @@ public void Reset ()
         /// </summary>
         [KRPCProperty]
         public IList<ScienceData> Data {
-            get { return experiment.GetData ().Select (data => new ScienceData (experiment, data)).ToList (); }
+            get { return experiment.GetData().Select(data => new ScienceData(experiment, data)).ToList (); }
+        }
+
+        /// <summary>
+        /// Determines if the experiment is available given the situationMask and requireAtmosphere.
```
Comment (djungelorm) [inline service/SpaceCenter/src/Services/Parts/Experiment.cs]: What are situationMask and requireAtmosphere? Does this property just return whther the experiment is available given the current conditions?

File: service/SpaceCenter/src/Services/Parts/ScienceSubject.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        /// <summary>
+        /// Amount of science already earned from this subject, not updated until after transmission/recovery.
+        /// </summary>
+        [KRPCProperty]
+        public float Science {
+            get { return data.science  * gainMultiplier; }
+        }
+
+        /// <summary>
+        /// Total science allowable for this subject, based on subjectValue.
```
Comment (djungelorm) [inline service/SpaceCenter/src/Services/Parts/ScienceSubject.cs]: What do you mean "based on subjectValue"???

File: service/SpaceCenter/src/Services/Parts/ScienceSubject.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        /// <summary>
+        ///  Whether the experiment has been completed.
+        /// </summary>
+        [KRPCProperty]
+        public bool IsComplete {
+            get { return ScienceCap - Science == 0; }
+        }
+
+        /// <summary>
+        /// Multiply science value by this dataScale value to determine data amount in mits.
```
Comment (djungelorm) [inline service/SpaceCenter/src/Services/Parts/ScienceSubject.cs]: Drop the word dataScale?

**Comment by djungelorm** [comment]
Thanks for the changes! Just got some minor queries related to the docs comments - see my inline comments.

**Comment by djungelorm** [comment]
Also, ignore the travis failures. Looks like one of the project dependencies has moved...

**Comment by Croissong** [comment]
I already tried adding ScienceSubject to ignores.txt but it still fails: [travis-ci](https://travis-ci.org/Croissong/krpc/builds/155662156)

**Comment by djungelorm** [comment]
You need to add ```SpaceCenter.ScienceSubject``` to doc/order.txt

```service/SpaceCenter/src/ignores.txt``` is unrelated to this error - that file is for ignoring Gendarme warnings which aren't actually run by Travis.

**Comment by djungelorm** [comment]
Also, it thinks "mits" is a spelling mistake. So you'll also need to add that word to ```doc/src/dictionary.txt```. Then it should all compile and test correctly! At least it does locally for me with that change.

**Comment by djungelorm** [comment]
Thanks!

**Pull request merged**